assets/js/daily_overview/constants.js

```diff
@@ -1,8 +1,35 @@
 import React from "react";
-import { READABLE_TIME_FORMAT } from "../constants/dates_and_times";
+import {
+  READABLE_DATE_TIME_FORMAT,
+  READABLE_TIME_FORMAT
+} from "../constants/dates_and_times";
 import moment from "moment";
 import { Link } from "react-router-dom";
 import { getSupplementOverviewURLFromUUID } from "../routing/routing_utils";
+import { JSON_POST_AUTHORIZATION_HEADERS } from "../constants/requests";
+import { SUPPLEMENT_EVENTS_RESOURCE_URL } from "../constants/urls";
+
+const DeleteSupplementFromDailyOverview = ({ uuid, supplement_name, time }) => {
```

Comment:
It's comical how shitty your Javascript used to be. Probably still are, but less comical.